src/coreimage.cs

```diff
@@ -441,6 +441,16 @@ interface CIContext_ImageRepresentation {
 		[return: NullAllowed]
 		NSData GetHeifRepresentation (CIImage image, CIFormat format, CGColorSpace colorSpace, CIImageRepresentationOptions options);
 
+		[iOS (15,0), Mac (12,0), MacCatalyst (15,0), TV (15,0)]
+		[Export ("HEIF10RepresentationOfImage:colorSpace:options:error:")]
+		[return: NullAllowed]
+		NSData GetHeif10Representation (CIImage image, CGColorSpace colorSpace, NSDictionary<NSString, NSObject> options, [NullAllowed] out NSError errorPtr);
+
+		// [iOS (15,0), Mac (12,0), MacCatalyst (15,0), TV (15,0)]
+		// [Wrap ("GetHeif10Representation (This, image, colorSpace, options.GetDictionary ()!, out error)")]
+		// [return: NullAllowed]
+		// NSData GetHeif10Representation (CIImage image, CGColorSpace colorSpace, NSDictionary<NSString, NSObject> options, [NullAllowed] out NSError errorPtr);
```

Comment:
do I need to comment in these wrappers? haven't hit any issues yet, but similar existing methods (see below) have them

Reply:
I added them in as comments, because I wanted to ask about them / thought I might need to add them

Reply:
they should be uncommented
the wrapper let developer uses the `CIImageRepresentationOptions` _strongly-typed_ class instead of the _weakly-typed_ `NSDictionary`